freesound/settings.py

```diff
@@ -259,6 +259,13 @@
 # local settings if needed ;)
 DATA_URL = "/data/"
 
+# Locations where sounds and previews will be mirrored (if specified)
+# If locations do not exist, they will be created
+MIRROR_DISK_LOCATIONS = {
+    'SOUNDS': None,  # mirrors contents of SOUNDS_PATH
+    'PREVIEWS': None,  # mirrors contents of PREVIEWS_PATH
+}
+
```

Comment:
(I see now the implementation does nothing if the data is None. Are you happy with that to turn on and off? If so, document this in the comments